project/standard/templates/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM tomcat:9-jdk11 AS mother
+LABEL maintainer="Alessandro Parma<alessandro.parma@geosolutionsgroup.com>"
+ARG MAPSTORE_WEBAPP_SRC="https://github.com/geosolutions-it/MapStore2/releases/latest/download/mapstore.war"
```

Comment:
same here, file downloads the latest mapstore, so dockerfile is unuseful in this case.
(2 copies of same docker file, too).

Reply:
will remove this file